frameworks/agno/finance-agent/agents/FinanceAgent/agent.py

```
@@ -0,0 +1,70 @@
+    }
+
+async def run(request: AgentRequest, response: AgentResponse, context: AgentContext):
+    try:
+        user_input = await request.data.text()
+        context.logger.info(f"Received financial analysis request: {user_input}")
+        
+        result = finance_agent.run(user_input)
+        
+        return response.text(str(result))
+        
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)